docs/en/enterprise-edition/policy-reference/azure-policies/azure-networking-policies/bc-azure-236.adoc

```diff
@@ -0,0 +1,69 @@
+
+== Azure Cognitive Services account configured with local authentication
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/resource/azure/CognitiveServicesEnableLocalAuth.py[CKV_AZURE_236]
+
+|Severity
+|LOW
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy is checking to ensure that local authentication is enabled for Azure Cognitive Services accounts. Failing to enable local authentication can expose your systems to unnecessary risks because it allows any authenticated user to make changes, even if they are not physically present in a secure, trusted environment. This can lead to unauthorized access, data breaches, and other security issues. Enabling local authentication adds an extra layer of security by ensuring that users are not only authenticated, but also accessing the system from a trusted location.
```

Comment:
This policy checks whether local authentication is disabled in Azure Cognitive Services accounts. Local authentication, when enabled, allows any authenticated user to make changes, regardless of location. This can be risky because it doesn't enforce additional security measures beyond basic authentication, and can potentially lead to unauthorized access, data breaches, and other security issues. Disabling local authentication strengthens security by requiring users to utilize more secure methods.